plugin/suda.vim

```diff
@@ -6,3 +6,17 @@ let g:loaded_suda = 1
 if get(g:, 'suda_startup', 1)
   call suda#init()
 endif
+
+if exists('g:suda_hijack_write')
```

Comment:
may

Using `get` is better because users can control on/off by the value like

```vim
if get(g:, 'suda_hijack_write')
    " ...
endif
```

```vim
let g:suda_hijack_write = 0  " Disable
" let g:suda_hijack_write = 1  " Enable
```